tests/unit/v1/test_query.py

```diff
@@ -1104,9 +1109,57 @@ def test_get_simple(self):
             metadata=client._rpc_metadata,
         )
 
-        # Verify the deprecation
-        self.assertEqual(len(warned), 1)
-        self.assertIs(warned[0].category, DeprecationWarning)
+    def test_get_limit_to_last(self):
+        from google.cloud import firestore
+
+        # Create a minimal fake GAPIC.
+        firestore_api = mock.Mock(spec=["run_query"])
+
+        # Attach the fake GAPIC to a real client.
+        client = _make_client()
+        client._firestore_api_internal = firestore_api
+
+        # Make a **real** collection reference as parent.
+        parent = client.collection("dee")
+
+        # Add a dummy response to the minimal fake GAPIC.
+        _, expected_prefix = parent._parent_info()
+        name = "{}/sleep".format(expected_prefix)
+        data = {"snooze": 10}
+        data2 = {"snooze": 20}
+
+        response_pb = _make_query_response(name=name, data=data)
+        response_pb2 = _make_query_response(name=name, data=data2)
+
+        firestore_api.run_query.return_value = iter([response_pb, response_pb2])
+
+        # Execute the query and check the response.
+        query = self._make_one(parent)
+        query = query.order_by(
+            u"snooze", direction=firestore.Query.DESCENDING
+        ).limit_to_last(2)
+        get_response = query.get()
+
+        self.assertIsInstance(get_response, list)
+        returned = list(get_response)
+        self.assertEqual(len(returned), 2)
+
+        snapshot = returned[0]
+        self.assertEqual(snapshot.reference._path, ("dee", "sleep"))
+        self.assertEqual(snapshot.to_dict(), data2)
+
+        snapshot2 = returned[1]
+        self.assertEqual(snapshot2.reference._path, ("dee", "sleep"))
+        self.assertEqual(snapshot2.to_dict(), data)
```

Comment:
Checking that data came in reverse order, as limit to last is a limit made from the end

Reply:
Just to make sure it's working as advertized, maybe it's worth adding another method to the system test as well.